### PR TITLE
tac_plus: fix compilation with GCC14

### PR DESCRIPTION
**Pull request by neheb**
Missing declaration and wrong pointer type.

Maintainer: @brunompena maybe

### CODE REVIEW

File: net/tac_plus/patches/020-gcc14.patch [hunk 1]
```diff
@@ -0,0 +1,30 @@
+--- a/tacacs-F4.0.4.28/do_author.c
++++ b/tacacs-F4.0.4.28/do_author.c
+@@ -30,6 +30,8 @@
+ # endif
+ #endif
+ 
++int cfg_get_logauthor(void);
```
Comment (brunompena) [inline net/tac_plus/patches/020-gcc14.patch:7]: This declaration would perhaps be better placed on **tac_plus.h:421**

File: net/tac_plus/patches/020-gcc14.patch [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
++
+ static int arg_ok(char *);
+ static char *assemble_args(struct author_data *);
+ static int authorize_cmd(char *, char *, struct author_data *);
+@@ -46,7 +48,6 @@ static int ppp_lcp_allowed(int, char *,
+ static int pre_authorization(char *, struct author_data *);
+ static char *value(char *);
+ 
+-
```
Comment (brunompena) [inline net/tac_plus/patches/020-gcc14.patch:16]: Typo? (unneeded change)

**Comment by brunompena** [comment]
It has been a long time since I submitted that package... thank you for fixing it!

File: net/tac_plus/patches/020-gcc14.patch (hunk 2 above)
Comment (neheb) [reply net/tac_plus/patches/020-gcc14.patch]: Fixed.

File: net/tac_plus/patches/020-gcc14.patch (hunk 1 above)
Comment (neheb) [reply net/tac_plus/patches/020-gcc14.patch]: Looks like it compiles there.

**Pull request merged**